packages/terra-clinical-error-view/lib/ErrorView.scss

```diff
@@ -0,0 +1,27 @@
+@import '~terra-mixins';
+
+.terraClinical-ErrorView {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  padding: 40px 0 20px;
+  width: 100%;
+}
+
+.terraClinical-ErrorView-text {
+  @include terra-text-align-center();
+  color: $terra-grey-25;
+  flex: 0 0 auto;
+  font-size: 1.7142857142857142rem;
+  line-height: 1.284;
+  padding-top: 4px;
+}
+
+.terraClinical-ErrorView-glyph {
+  flex: 0 0 auto;
+}
+
+.terraClinical-ErrorView-button {
+  flex: 0 0 auto;
+  padding-top: 4px;
```

Comment:
Should we keep units consistent? I think rem is the only unit being used in terra-core